hazelcast-jet-core/src/main/java/com/hazelcast/jet/Processors.java

```diff
@@ -1128,30 +1136,10 @@ protected boolean tryProcess(int ordinal, @Nonnull Object item) throws Exception
         }
     }
 
-    /**
-     * Processor with the following semantics:
-     * <ul><li>
-     *     Accepts items of type {@code T}.
-     * </li><li>
-     *     Computes the key of type {@code K} by applying the key extractor
-     *     to the item.
-     * </li><li>
-     *     Maintains a set of all seen keys.
-     * </li><li>
-     *     Emits the size of the set (the number of seen distinct keys) as a
-     *     {@code Long} value.
-     * </li></ul>
-     *
-     * @param <T> type of received item
-     * @param <K> type of grouping key
-     */
```

Comment:
This Javadoc should not be removed, it documents clearly what the processor does. Instead it should be moved to the factory method and adapted. See `groupAndAccumulate` for an example.